### PR TITLE
Temp push

### PR DESCRIPTION
**Pull request by narsiiiii**

### CODE REVIEW

File: src/boxComponents/CardList.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from '@spraoi/base/Box';
+
+const CardList = ({ children }) => {
+  const rest = {
```
Comment (cadejscroggins) [inline src/boxComponents/CardList.js]: These rest variables are unnecessary. Let's pass these as normal props.

File: src/boxComponents/IconCardList.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from '@spraoi/base/Box';
+import './styles/iconCardList.css';
+
+const IconCard = ({ title, description, icon, list }) => {
+  const listItems = list.reduce((res, row, ix) => {
```
Comment (cadejscroggins) [inline src/boxComponents/IconCardList.js]: I think `Array.prototype.map` makes more sense here.

File: src/boxComponents/SectionH1.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+// position: relative;
```
Comment (cadejscroggins) [inline src/boxComponents/SectionH1.js]: Do we need this comment?

File: src/data/dataPages/ML.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
+const ML = {
```
Comment (cadejscroggins) [inline src/data/dataPages/ML.js]: Is this data used in more than one place? If not, it can be moved to the component that uses it.

Comment (narsiiiii) [reply src/data/dataPages/ML.js]: Removed this file

File: src/boxComponents/SectionH1.js (hunk 3 above)
Comment (narsiiiii) [reply src/boxComponents/SectionH1.js]: This was used to replicate existing styled components to Box. Forgot to remove it after modification

File: src/boxComponents/CardList.js (hunk 1 above)
Comment (narsiiiii) [reply src/boxComponents/CardList.js]: Done

**Comment by narsiiiii** [comment]
> Looks good @narsiiiii. Couple requests.
> 
> * `.DS_Store` files should be in your [global gitignore](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer).
> * For consistency, let's use the same directory structure as the rest of the components (i.e `src/components/SomeComponent/index.js` vs having a separate `boxComponents` directory).
> * We shouldn't be writing any `.css` files. Are those necessary?
Required for the small animations that were introduced.
> * Tests need to pass.

**Pull request merged**